herd/AArch64Arch_herd.ml

```diff
@@ -399,11 +401,17 @@ module Make (C:Arch_herd.Config)(V:Value.AArch64) =
       | I_RDVL (r,_) | I_ADDVL (r,_,_) | I_CNT_INC_SVE (_,r,_,_)
       | I_LD1SPT (_,r,_,_,_,_,_) | I_MOVA_TV (r,_,_,_,_) | I_MOVA_VT (r,_,_,_,_)
       | I_ADDA (_,r,_,_,_)
+      | I_GCSSTR (r,_)
         -> [r]
       | I_LDAR (_,(XX|AX),r,_) |I_LDARBH (_,(XX|AX),r,_)
         -> [r;ResAddr;]
       | I_STXR (_,_,r,_,_) | I_STXP (_,_,r,_,_, _) | I_STXRBH (_,_,r,_,_)
         -> [r;ResAddr;]
+      | I_BL _ | I_BLR _ | I_RET _
```

Comment:
Is it easy to check if variant shadowstack has been enabled? It's not a huge problem to "kill" GCSPR_EL1 when not needed, so just checking.